### PR TITLE
Add script to ease changelog updates

### PR DESCRIPTION
**Pull request by vtnate**
This script was copied from https://github.com/NREL/OpenStudio-server/blob/master/lib/change_log.rb (I made a few very minor changes that don't alter behavior).
The original file from OpenStudio-server lives in the /lib directory. I put this one in the same place in this repo, but wonder if that's the appropriate place for it. Would love an opinion on that.

### CODE REVIEW

**Comment by vtnate** [comment]
I set the github_api dependency in the gemspec as a normal dependency and not as a development dependency, primarily because I don't understand the distinction between those two. I'd love some education on what cases to use one or the other.

**Comment by vtnate** [comment]
Now that I look at it, https://github.com/NREL/OpenStudio-server/tree/master has no *.gemspec file. Now I really don't know what's going on.

File: openstudio-extension.gemspec [hunk 1]
```diff
@@ -27,6 +27,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'openstudio-workflow', '~> 1.3.4'
   spec.add_dependency 'openstudio_measure_tester', '~> 0.1.7'
   spec.add_dependency 'parallel', '~> 1.12.0'
+  spec.add_dependency "github_api", "~> 0.18.0"
```
Comment (nllong) [inline openstudio-extension.gemspec]: @vtnate  -- I'm concerned about adding the gem dependency. As far as I can remember OpenStudio will need to be able to load this Gem (and dependent gems). I need to verify this with @kbenne

Comment (vtnate) [reply openstudio-extension.gemspec]: I hear ya. There are definitely some things I don't yet understand about Ruby packages. As I mentioned above, OpenStudio-server doesn't have a `gemspec` file, but its `change_log.rb` file requires `github_api`. What am I missing?

**Comment by kflemin** [comment]
@nllong how about if we put github-api as a development dependency? is that cool?

**Comment by nllong** [comment]
well, looks like that happened. If it is a dev dependency, then all should be good.

**Pull request merged**